shop.py

```diff
@@ -0,0 +1,26 @@
+def shop():
+    customers_available_money = 100.00
+    items_dict = {'Balenciaga Trainers': 785.00, 'Northface Jacket': 320.00, 'Gucci Handbag': 850.00}
+
+    print("Welcome, Check out the designer items I have and their prices:")
+    for item, price in items_dict.items():
+        print(f"{item}: £{price}")
+
+    while True:
+        designer_item = input("Enter the designer item you would like to purchase or type 'exit' to leave the shop: ")
+
+        if designer_item == 'exit':
+            print("Thank you for shopping with us!")
+            break
+
+        if designer_item not in items_dict:
+            print("Invalid item choice, please try again.")
+            continue
+
+        if items_dict[designer_item] > customers_available_money:
+            print(f"You cannot afford {designer_item}, please enter a new item or type 'exit' to leave.")
+            continue
+
+        customers_available_money -= items_dict[designer_item]
+        print(f"Here's your {designer_item}!")
+shop()
```

Comment:
7/14 => Missed some of the criteria for what the shop should be able to do. Here is a checklist 

 1. Create a dictionary with a minimum of 3
items and prices
a. One of the items needs to cost
more than £100

2. Customer’s available money is £100
3. Welcome the customer and display the
items and their prices, along with an
option to “exit”
4. Accept the option as an input, an invalid
input should raise a ValueError
5. If the customer can afford it, print out a
message saying “Here’s your {item}!”
6. The user should then be greeted out of
the shop, and the program terminated.
7. If the customer cannot afford it, note the
attempt and ask if they have more money,
if they do and enter the amount it should
be added to the balance.
8. The purchase should be tried a maximum
of 3 items, if it fails a custom error should
be raised and the customer will exit the
shop.

Also no try/catch block for exception handling